grakn-bootup/src/main/java/ai/grakn/bootup/Graql.java

```diff
@@ -103,7 +104,7 @@ private void migrate(String[] args) {
                 JsonMigrator.main(valuesFrom(args, 1));
                 break;
             case "owl":
-                System.out.println("Owl migration not supported anymore");
+                System.err.println(ErrorMessage.OWL_NOT_SUPPORTED.getMessage());
```

Comment:
so why don't we remove it completely? BO reasons?

Reply:
I think it's good to keep, as there are people coming over from OWL asking about if they can migrate automatically